knimin/templates/pm_plate_list.html

```diff
@@ -0,0 +1,173 @@
+<link rel="stylesheet" type="text/css" href="/static/vendor/css/jquery.dataTables.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
+<style>
+  p {
+    font-weight: bold;
+  }
+  a {
+    color: black;
+  }
+  button {
+    padding: 5px 10px 5px 10px;
+    text-decoration: none;
+    font-size: 80%;
+  }
+  select {
+    border: 1px solid black;
+    padding: 5px 10px 5px 10px;
+    height: 25px;
+    margin-bottom: 1px;
+  }
+  button.inline {
+    height: 25px;
+    width: 25px;
+    padding: 2px 4px 2px 4px;
+    border: 0;
+    background-color: transparent;
+    text-decoration: none;
+    font-size: 80%;
+  }
+  button.inline:hover {
+    background-color: dimgrey;
+    color: white;
+  }
+  table {
+    border-top: 1px solid black;
+  }
+  table.dataTable thead th {
+    text-align: left;
+    padding: 5px 10px 5px 10px;
+  }
+  tbody.list tr td {
+    vertical-align: middle;
+    padding: 5px 10px 5px 10px;
+  }
+  tbody.list tr:hover {
+    background-color: lightgray;
+  }
+  tbody.list a:hover {
+    color: darkblue;
+    text-decoration: underline;
+  }
+  .dataTables_wrapper .dataTables_info {
+    font-size: 90%;
+  }
+  .dataTables_wrapper .dataTables_filter {
+    font-size: 90%;
+  }
+  .dataTables_wrapper .dataTables_length {
+    font-size: 90%;
+  }
+  .dataTables_wrapper .dataTables_paginate {
+    font-size: 90%;
+  }
+  .dataTables_wrapper .dataTables_paginate .paginate_button {
+    padding: 4px 8px 4px 8px;
+    text-decoration: none;
+      /* padding : 0px;
+      margin-left: 0px;
+      display: inline;
+      border: 0px; */
+  }
+  .dataTables_wrapper .dataTables_paginate .paginate_button:hover {
+      /* border: 0px; */
+  }
+</style>
+<script type="text/javascript">
+  $(document).ready(function(){
+    // var plates = {#plates#};
+    $("#btnNew").click(function(){
+        // alert("button");
+        alert(this.innerHTML);
+        alert($(this).text());
+    });
+    $('#plate_list').DataTable({
+      scrollY: 450,
+      scrollCollapse: true,
+      pageLength: 50
+    });
+    $('#plate_list tbody').on('click', 'tr', function(){
+        $(this).toggleClass('selected');
+    });
+  });
+  window.onload = function() {
+    // items=items, pageno=pageno, total=total, plates=plates
+  }
+</script>
+{% end %}
+
+{% block content %}
+<h3 id="page_title">{{target[0].upper() + target[1:]}} Plate List</h3>
+<p>
+  <!--span style="float:right"><button id="btnNew">Create new plate</button></span-->
+  <button id="btnNew"><i class="fa fa-plus"></i> Create new</button>
+  <button id="btnDel"><i class="fa fa-remove"></i> Delete selected</button>
+  <button id="btnEdit"><i class="fa fa-edit"></i> Batch-edit selected</button>
+  <button id="btnDNA"><i class="fa fa-flask"></i> Convert to DNA plate</button>
+  <button id="btnEdit"><i class="fa fa-download"></i> Export list</button>
```

Comment:
@qiyunzhu I don't really care which way we go as long as it is consistent. The comment from @ElDeveloper points out that you sometimes are using camel-case (`btnNew`) and sometimes you're using underscore separated (`page_tile`). Consistency is important to facilitate code development. I took a look around this project and there is no consistency in general, so that's why I don't have strong feelings to which way go as long as all the files that you're adding right now use the same.

In Qiita we've been using dash separated, and as per @ElDeveloper comment I think they used the same in Emperor.

Also, note that the Google closure document is pointing out JavaScript variables, not DOM ids.

@ElDeveloper do you have strong feelings to which way to go? I don't but I would like to enforce consistency.